src/providers/be_dyndns.c

```diff
@@ -1217,6 +1217,7 @@ static struct dp_option default_dyndns_opts[] = {
     { "dyndns_update_ptr", DP_OPT_BOOL, BOOL_TRUE, BOOL_FALSE },
     { "dyndns_force_tcp", DP_OPT_BOOL, BOOL_FALSE, BOOL_FALSE },
     { "dyndns_auth", DP_OPT_STRING, { "gss-tsig" }, NULL_STRING },
+    { "dyndns_auth_ptr", DP_OPT_STRING, { "" }, NULL_STRING },
```

Comment:
Hi,

I think it would be more consistent with other options to use `NULL_STRING` and check for NULL or the empty string in `be_nsupdate_init`.

bye,
Sumit